satpy/readers/core/hrit.py

```diff
@@ -312,7 +311,6 @@ def __init__(self, filename, mda):
         self.bpp = mda["number_of_bits_per_pixel"]
         self.compressed = mda["compression_flag_for_data"] == 1
         self.offset = mda["total_header_length"]
-        self.zipped = os.fspath(filename).endswith(".bz2")
```

Comment:
This `self.zipped` just isn't being used anywhere?

Reply:
yes, looked like it.